webui_runner/plugs/wasmtime_app/echo_branch.sh

```diff
@@ -0,0 +1 @@
+echo master
```

Comment:
Do we want to figure this out dynamically in case someone is doing some experiments with a branch?